spec/support/request_helpers/downstream_requests.rb

```diff
@@ -4,18 +4,16 @@ def sends_to_draft_content_store
       it "sends to draft content store" do
         allow(PublishingAPI.service(:draft_content_store)).to receive(:put_content_item).with(anything)
 
-        Timecop.freeze do
-          expect(PublishingAPI.service(:draft_content_store)).to receive(:put_content_item)
-            .with(
-              base_path: base_path,
-              content_item: content_item_for_draft_content_store
-                .merge(transmitted_at: DateTime.now.to_s(:nanoseconds))
-            )
-
-          do_request
-
-          expect(response.status).to eq(200), response.body
-        end
+        expect(PublishingAPI.service(:draft_content_store)).to receive(:put_content_item)
+          .with(
+            base_path: base_path,
+            content_item: content_item_for_draft_content_store
+              .merge(payload_version: anything)
+          )
+
+        do_request
+
+        expect(response.status).to eq(200), response.body
```

Comment:
`expect(response.status).to be_ok, response.body` might read nicer but to be honest, this style of multiple expectations on a single line makes me have to stop and think every time. :speak_no_evil: